### PR TITLE
Parser

### PR DESCRIPTION
**Pull request by mezgoodle**

### CODE REVIEW

**Comment by mezgoodle** [comment]
CI **will not work** for this _branch_, due to [workflow](https://github.com/mezidia/song-helper/blob/django/.github/workflows/django.yml) in _django_ branch. But I ran tests locally and give you result below:
image

File: server/mesite/tests.py [hunk 1]
```diff
@@ -61,3 +62,31 @@ def test_song_content(self):
...
+class SearcherTest(TestCase):
+    """
+    Test search on YouTube
+    """
+    def test_searcher(self):
+        """
+        Test the function output
+        """
+        str_to_search = 'python'
+        params = {'limit': 1}
```
Comment (Dmytrenko-Roman) [inline server/mesite/tests.py:76]:
```suggestion
        params1 = {'limit': 1}
```

File: server/mesite/tests.py [hunk 2]
```diff
@@ -61,3 +62,31 @@ def test_song_content(self):
...
+        params_1 = {'limit': 3}
```
Comment (Dmytrenko-Roman) [inline server/mesite/tests.py:77]:
```suggestion
        params2 = {'limit': 3}
```

File: server/mesite/tests.py [hunk 3]
```diff
@@ -61,3 +62,31 @@ def test_song_content(self):
...
+        result = search_youtube(str_to_search, params)
```
Comment (Dmytrenko-Roman) [inline server/mesite/tests.py:78]: Rewrite all variables. Example: result & result_1 to result1 & result2.

File: server/requirements.txt [hunk 4]
```diff
@@ -1,4 +1,12 @@
...
 Django==3.1.6
+h11==0.12.0
+httpcore==0.12.3
+httpx==0.17.1
+idna==3.1
 pytz==2021.1
+rfc3986==1.4.0
+sniffio==1.2.0
 sqlparse==0.4.1
+youtube-search-python==1.4.2
```
Comment (Dmytrenko-Roman) [inline server/requirements.txt:12]: where is nodejs?

**Review by Dmytrenko-Roman — CHANGES_REQUESTED** [review]

**Review by Dmytrenko-Roman — APPROVED** [review]

File: server/requirements.txt (hunk 4 above)
Comment (mezgoodle) [reply server/requirements.txt:12]: On my computer

**Pull request merged**